src/OrbitBase/Logging.cpp

```diff
@@ -10,15 +10,32 @@
 #include <absl/strings/str_format.h>
 #include <absl/synchronization/mutex.h>
 #include <absl/time/time.h>
+#include <stdio.h>
 
 #include <array>
 #include <system_error>
 
 #include "LoggingUtils.h"
+#include "OrbitBase/SafeStrerror.h"
 #include "OrbitBase/ThreadUtils.h"
+#include "OrbitBase/UniqueResource.h"
+
+namespace orbit_base {
+
+#if !defined(TEMP_FAILURE_RETRY)
+// Windows does not have TEMP_FAILURE_RETRY, define it here
+#define TEMP_FAILURE_RETRY(expression)                 \
+  do {                                                 \
+    int64_t result = static_cast<int64_t>(expression); \
+  } while (result == -1 && errno == EINTR)
```

Comment:
```suggestion
#define TEMP_FAILURE_RETRY(expression)           \
  do {                                           \
    int64_t result;                              \
    do {                                         \
      result = static_cast<int64_t>(expression); \
    } while (result == -1 && errno == EINTR);    \
  } while (0)
#endif
```

Reply:
removed this, now we have shortcut to expression instead.